### DISCUSSION TITLE
Issue while logging in with facebook

### DISCUSSION THREAD

**Discussion by RitikDeqode** [cuongdevjs/reactjs-social-login#108] [Q&A] [answered]
Hi, 
I am using your package for social media login for react project.
Gmail logging in is working fine and I could log in but with Facebook getting errors. (Attaching the screenshot) 
![facebookLoginError](https://github.com/cuongdevjs/reactjs-social-login/assets/79850780/e38742db-57e5-4005-99d6-848f1d788917)
I tried replicating the same code of your code sandbox and replacing the Facebook app id with mine but still facing the issue.
I am serving the app https, and updated the developer config as well with the URL.
Can you please help me if I am doing anything wrong?
and can you please also tell key points while configuring the config/.env as well

**Comment by cuongdevjs** [comment]
Please provide more information about your code.

**Reply by cuongdevjs** [reply, marked as answer]
If you running your app locally on your computer, maybe it fails, please try to serve your site as https and then try again.
